pkgs/by-name/at/atproto-goat/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+buildGoModule {
+  pname = "atproto-goat";
+  version = "0-unstable-2024-10-08";
+
+  src = fetchFromGitHub {
+    owner = "bluesky-social";
+    repo = "indigo";
+    rev = "06bacb465af714feb77609566aba15ab1ed41e24";
+    # To get proper version information
+    leaveDotGit = true;
+    hash = "sha256-p6zSnK0pbo4nO6Si9om6qgbi4t4Xr8qZVe+j/E3ca0g=";
+  };
+
+  vendorHash = "sha256-T+jtxubVKskrLGTUa4RI24o/WTSFCBk60HhyCFujPOI=";
+
+  subPackages = [ "cmd/goat" ];
+
+  passthru.updateScript = unstableGitUpdater {
+    hardcodeZeroVersion = true;
+  };
+
+  meta = {
+    description = "Go AT protocol CLI tool";
+    homepage = "https://github.com/bluesky-social/indigo/blob/main/cmd/goat/README.md";
+    license = with lib.licenses; [
+      mit
+      asl20
+    ];
+    maintainer = with lib.maintainers; [ pyrox0 ];
+    mainProgram = "goat";
+    platforms = lib.platforms.all;
```

Comment:
```suggestion
```

This is not needed as `buildGoModule` adds platforms available for golang.

Reply:
Fixed